Mathlib/Data/Finsupp/Fin.lean

```diff
@@ -92,4 +92,11 @@ theorem cons_ne_zero_iff : cons y s ≠ 0 ↔ y ≠ 0 ∨ s ≠ 0 := by
   rw [h', c, Finsupp.cons_zero_zero]
 #align finsupp.cons_ne_zero_iff Finsupp.cons_ne_zero_iff
 
+lemma cons_support : (s.cons y).support ⊆ insert 0 (s.support.map (Fin.succEmb n).toEmbedding) := by
```

Comment:
Do we have a similar lemma about `Fin.cons`? IMHO, lemmas about `Finsupp.cons` should be `mod_cast (lemma about Fin.cons)`, if possible.

Reply:
What would the lemma about `Fin.cons` be? The ingredients in the statement seem quite `Finsupp`-specific.